src/OpenLoco/src/Vehicles/VehicleHead.cpp

```diff
@@ -1057,7 +1059,7 @@ namespace OpenLoco::Vehicles
         status = Status::stopped;
         vehType2 = _vehicleUpdate_2;
 
-        if (vehType2->var_73 & Flags73::isBrokenDown)
+        if ((bool)(vehType2->var_73 & Flags73::isBrokenDown))
```

Comment:
Why not do something like instead:
```suggestion
        if ((vehType2->var_73 & Flags73::isBrokenDown) != Flags73::none)
```
C style casts are not optimal

Reply:
Oh, didnt knew that was a thing, will fix.